src/bacnet/basic/service/h_wpm.c

```diff
@@ -83,9 +83,9 @@ static int write_property_multiple_decode(
                                      ERROR_CLASS_PROPERTY) &&
                                     (wp_data->error_code ==
```

Comment:
if i were you i'd have used the #if BTL_TEST_9_23_2_X5_workaround around entire if clause here.